atomic_reactor/plugins/pre_fetch_sources.py

```diff
@@ -301,6 +301,35 @@ def _get_kojifile_source_urls_helper(self, koji_build):
 
         return sources
 
+    def _get_remote_file_urls_helper(self, koji_build):
+        """Fetch remote source file urls from specific build
+
+        :param koji_build: dict, koji build
+        :return: str, URL pointing to remote source files
+        """
+
+        self.log.debug('get remote_file_urls: %s', koji_build['build_id'])
+
+        archives = self.session.listArchives(koji_build['build_id'], type='remote-source-file')
+        self.log.debug('archives: %s', archives)
+
+        remote_source_files_path = self.pathinfo.typedir(koji_build, btype='remote-source-file')
+        sources = []
+
+        for archive in archives:
+            if archive['type_name'] == 'tar':
+                # download each remote-source-file archive into it's own subdirectory
+                #  with the same name as the archive
+                source = {'url': os.path.join(remote_source_files_path,
+                                              archive['filename']),
+                          'subdir': archive['filename'].split('.')[0],
```

Comment:
I am not sure which extensions that file has but  I guess there can be dots in the filename as well
so maybe we could do rsplit('.', 1) ?

Reply:
hmm, I think we'll have to use rsplit('.', 2) in that case since we're getting a lot of .tar.gz files.

Reply:
updated.